services/ui-src/src/components/app/App.tsx

```diff
@@ -13,9 +12,17 @@ import {
   SkipNav,
   Sidebar,
 } from "components";
+// utils
+import { useUser } from "utils";
+
+export const SidebarOpenContext = createContext({
+  sidebarIsOpen: true,
+  setSidebarIsOpen: (status: boolean) => {}, // eslint-disable-line @typescript-eslint/no-unused-vars
+});
```

Comment:
I keep going back and forth on if this should be in it's own component or not. Fine where it is but I want my struggle to be known.

Reply:
I feel the same way. I think I'd prefer to keep it close the its use case for now, though I see where you're coming from.